exercises/J-array-get-set/exercises2.js

```diff
@@ -7,6 +7,7 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers.push(4);
```

Comment:
well done for using the `.push()` method. I think this exercise wanted you to practice setting values in an array using the square bracket notation, i.e. `myArray[0]`
Can you add the number 4 to the array using this notation?